docs/pages/access-controls/access-request-plugins/access-monitoring-rules.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Routing Access Requests to Access Plugins
+description: How to set up Teleport's Access Monitoring Rules
+---
+
+With Teleport's Access Monitoring Rules access request notifications
+can be routed to plugins based on based on several conditions.
+
+Currently only the Slack access plugin is supported.
+
+## Prerequisites
+
+- A Teleport Enterprise Cloud account.
+
+- The Enterprise `tctl` admin tool and `tsh` client tool version >= (=teleport.version=).
+
+  You can verify the tools you have installed by running the following commands:
+
+  ```code
+  $ tctl version
+  # Teleport Enterprise v(=teleport.version=) go(=teleport.golang=)
+  
+  $ tsh version
+  # Teleport v(=teleport.version=) go(=teleport.golang=)
+  ```
+
+  You can download these tools by following the appropriate [Installation 
+  instructions](../../installation.mdx#linux) for your environment and Teleport edition.
+
+- At least of the Teleport Access Plugins that support using being set up.
+
+- (!docs/pages/includes/tctl.mdx!)
+
+## Step 1/2. Creating an Access Monitoring Rule.
+
```

Comment:
I would introduce this example with text above the snippet, since otherwise it appears to arrive with no context.